README.md

```diff
@@ -180,8 +187,8 @@ class StaticServer(BaseServer):
             self._handler_stats_callback)
 
 def main():
-    server = StaticServer(ip='', port='1069', retries=3, timeout=5,
```

Comment:
i don't understand why you had to do this, I mean, isn't it clear from the StaticServer function what each argument is? :D